src/containers/Login/Login.js

```diff
@@ -30,27 +31,21 @@ class Login extends Component {
     let { name, value } = e.target;
     this.setState({ [name]: value })
   }
-
+  
   render() {
-    const {email, password} = this.state
+    const inputFields = Object.keys(this.state).map(field => {
```

Comment:
Nice, I like how dynamic this is if we ever needed to add additional input fields.

Reply:
👍 It's not 'pretty' to look at, however it makes it super easy to update and not have to worry about later